src/reduce.js

```diff
@@ -123,3 +122,60 @@ export function reduceSum(input, options = {}) {
   return reduce(input,
       (previousValue, currentValue) => previousValue + currentValue, options);
 }
+
+/**
+ * Compute the sum of the square of all the input values along the axes.
+ * @param {Tensor} input
+ * @param {MLReduceOptions} options
+ * @return {Tensor}
+ */
+export function reduceSumSquare(input, options = {}) {
+  return reduceSum(pow(input, new Scalar(2)), options);
```

Comment:
Formulas all look correct, but wasn't there a GitHub issue with ReduceL2 ambiguity?

The formula I recall was ReduceL2 = `sqrt(a1^2 + a2^2 + ... + an^2)`.

(btw, I always have to re-lookup the formulas because I keep thinking "reduceSumSquare" means `x.sum().square()`, but actually it means `x.square().sum()` - so square and *then* reduce sum)

Reply:
> but wasn't there a GitHub issue with ReduceL2 ambiguity?

Are there other explanations for **ReduceL2** besides `ReduceL2 = sqrt(a1^2 + a2^2 + ... + an^2) `? 

Reference: https://en.wikipedia.org/wiki/Norm_(mathematics)#p-norm

Reply:
Hmm, I probably misremember with a different issue.

Reply:
Oh never mind, it was actually L2 pooling: https://github.com/webmachinelearning/webnn/issues/278